code/modules/events/flutes.dm

```diff
@@ -0,0 +1,142 @@
+
+/datum/round_event/flutes/proc/flute_chanting(mob/living/carbon/M)
+	to_chat(M, "<span class ='cultlarge'><b>You hear faint chanting.. You feel a heavy weight upon your shoulders, as something shifts it's gaze towards you..</b></span>")
+	M.blur_eyes(30)
+	M.adjustOrganLoss(ORGAN_SLOT_BRAIN, 50, 60)
+	M.adjustStaminaLoss(50)
+	ADD_TRAIT(M, TRAIT_UNSTABLE, M)
+	//sanity = 75
+
+/datum/round_event/flutes/proc/flute_starlight(mob/living/carbon/M)
+	to_chat(M, "<span class='warning'><b>As the nearest stars light your skin and your station, you can make out the faint whispers being spoken in turn with the monotone flutes playing beyond you. You feel so tired, as the struts of metal piping, the walls, the floor twist in unnatural ways, as the lights dim.</b></span>")
+	sleep(300)
+	M.blur_eyes(40)
+	M.adjustOrganLoss(ORGAN_SLOT_BRAIN, 65, 70, 80)
+	ADD_TRAIT(M, TRAIT_UNSTABLE, M)
+	//sanity = 50
+	sleep(250)
+	to_chat(M, "<span class='warning'><b>Exhaustion takes you.</b></span>")
+	M.SetSleeping(70)
+	to_chat(M, "<span class='notice'><b>You see vague, inhuman forms, in the tightest corners of your alien dreams, dancing frantically to thundering drums, in tune with yellow monotone flutes.</b></span>")
+	sleep(200)
+	M.adjustBruteLoss(15)
+	to_chat(M, "<span class='notice'><b>You wake from your deep dreams, blood oozing from your nose, while your head beats to a familiar tune.</b></span>")
+
+/datum/round_event/flutes/proc/flute_corridor_encounter(mob/living/carbon/M)
+	to_chat(M, "<span class='notice'><b>You glance upwards, down the corridor.</b></span>")
+	sleep(50)
+	to_chat(M, "span class='bold large_brass'><b>A BRIGHT BLINDING LIGHT FILLS YOUR VISION!</b></span>")
+	M.eye_blind = 5
+	sleep(100)
+	to_chat(M, "<span class='reallybig hypnophrase'><b>Your mind oozes at what it cannot comprehend, you feel your soul snatching onto your brain, with chills rolling up your spine. You barely hang on.</b></span>")
+	if(HAS_TRAIT(M, TRAIT_UNSTABLE))
+		to_chat(M, "<span class='green'><b>Your mind shatters.</b></span>")
+		M.SetSleeping(30)
+		sleep(300)
+		to_chat(M, "<span class='suicide'><b>Your flesh undulates, and boils off your bones. You were blind, yet now you've seen a glimpse behind the cosmic curtain.</b></span>")
+		//sanity = 25
+		REMOVE_TRAIT(M, TRAIT_UNSTABLE, M)
+		M.blur_eyes(5)
+		M.adjustStaminaLoss(90)
+		M.adjustBruteLoss(90)
+		M.adjustOrganLoss(ORGAN_SLOT_BRAIN, 90)
+		M.set_species(/datum/species/skeleton)
+		sleep(300)
+		to_chat(M, "<span class='warning'><b>Your mind.. Has been humbled. You have escaped by the skin of your teeth, but have lost your body, and much, much more. You feel.. Mirth?</b></span>")
+	else
+		to_chat(M, "span class='suicide'><b>Your mind shatters.</b></span>")
+		ADD_TRAIT(M, TRAIT_UNSTABLE, M)
+		//sanity = 1
+		M.SetSleeping(30)
+		M.blur_eyes(40)
+		M.adjustStaminaLoss(99)
+		to_chat(M, "<span class='Narsie'><b>Y'HAH HT'HU THRZHZU. UA'KLL GHRT AWN ZUU!</b></span>")
+		M.adjustBruteLoss(99)
```

Comment:
Very close to the crit point

Reply:
purposeful, I want an event that effects a small number of people, and can potentially really fuck them up. If you have a better idea, please tell.